ic-os/guestos/context/Dockerfile

```diff
@@ -209,6 +209,11 @@ RUN addgroup node_exporter && \
     chmod 0644 /etc/default/node_exporter \
                /etc/node_exporter/web.yml
 
+# The user to run the SOCKS proxy
+RUN addgroup socks && \
+    adduser --system --disabled-password --shell /usr/sbin/nologin -c "Dante SOCKS Proxy" socks && \
+    adduser socks socks && chmod +s /usr/sbin/danted
+
```

Comment:
Move this to the end of user/group additions so that the previous UID/GID stay the same when we build the IC-OS

Reply:
Done